source/release-notes.txt

```diff
@@ -10,6 +10,34 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.40.3
+----------------
+
+*Released October 11, 2023*
+
+New Features:
+
+- After creating a $search index you are now redirected to the search 
+  indexes modal (:issue:`COMPASS-7247`).
+- Field names are now autocompleted when defining indexes. (:issue:`COMPASS-7174`).
+- Syntax errors are now highlighted when defining indexes. (:issue:`COMPASS-7246`).
+- Improved AI feedback experience (:issue:`COMPASS-7211`, :issue:`COMPASS-7251`).
+- Removed ``useNewUrlParser`` and ``useUnifiedTopology`` from export 
+  to language options (:issue:`COMPASS-4897`).
+- The knnVector index template definition is now used instead of 
```

Comment:
(suggestion)
If you go with the suggestion above to group the index improvements under an Atlas Search bullet, this last tickets goes there. 

(suggestion -- active voice)
Consider a slight rewrite for active voice ...  something like:
The ``vectorEmbedding`` index template definition replaced ``knnVector``. [One sentence about the benefits of vectorEmbedding, perhaps.]